index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey form</title>
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    <div class="container">
+        <header class="heading">
+            <h1 id="title">freeCodeCamp Survey Form</h1>
+            <p class="survey-text" id="description">Thank you for the time to help us improve the platform</p>
+        </header>
+        <form action="" id="survey-form">
+            <div class="forminfo">
+                <label for="name" id="name-label">Name</label>
+                <input class="forms" type="text" id="name" placeholder="Enter your name" required>
+                
+            </div>
+            <div class="formifo">
+                <label for="email" id="email-label">E-mail</label>
+                <input  class="forms" type="email" id="email" name="email" placeholder="Enter your email" required>
+            </div>
+            <div class="forminfo">
+                <label for="Number" id="number-label">Age</label>
+                <input  class="forms" type="number" id="number" placeholder="Age" min="18" max="120">
+            </div>
+            <div class="forminfo">
+                <p>Whic option best describes your current role</p>
+                <select id="dropdown" name="role" class="forms" required>
+                    <option disabled selected value>Select current role</option>
+                    <option value="student">Student</option>
+                    <option value="work">Full Time Job</option>
+                    <option value="learning">Full Time Learner</option>
+                    <option value="notPrefered">Prefer not to say</option>
+                    <option value="other">Other</option>
+      
+                </select>
+            </div>
+            <div class="forminfo">
+                <p>Would you recommrn freeCodeCamp to a friend?</p>
```

Comment:
- [ ] Your message below is not clear as to what the issue is. Always ensure that your message is clear and easy to understand.

```
You are almost there, kindly add an unordered list to your dropdown
Select current role Student Full Time Job Full Time Learner Prefer not to say Other </select>
```

 For Example:
* This is a clear comment 👇🏾  
    - The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

* This is a vague comment 👇🏾 
    - The README is not descriptive, why don't you make it better?